package.json

```diff
@@ -97,21 +98,20 @@
   "scripts": {
     "lint": "eslint lib test",
     "test": "if [ $COVERAGE ]; then npm run coveralls; else npm run jshint && npm run teaser && npm run test-unit && npm run test-integration; fi",
-    "test-docker": "docker-compose run sequelize /bin/sh -c \"npm run test-all\"",
-    "test-docker-unit": "docker-compose run sequelize /bin/sh -c \"npm run test-unit-all\"",
-    "test-docker-integration": "docker-compose run sequelize /bin/sh -c \"npm run test-integration-all\"",
-    "build-docker": "docker-compose build",
+    "test-docker": "env-cmd ./test/config/.docker.env npm run test-all",
+    "test-docker-unit": "env-cmd ./test/config/.docker.env npm run test-unit-all",
+    "test-docker-integration": "env-cmd ./test/config/.docker.env npm run test-integration-all",
     "docs": "node docs/docs-generator.js",
-    "jshint": "./node_modules/.bin/jshint lib test",
```

Comment:
One thing I was confused about is that there is both jshint and eslint in this project. Are we trying to convert to eslint or sticking with jshint? jshint is used in `npm test`, but eslint is used in the `npm run lint` command...just curious.